teamprojekt_competition_server/client/main.py

```diff
@@ -43,5 +45,5 @@ def step(self, obv: list[float]) -> list[float]:
             return [random.choice([0.0, 1.0, 2.0])]
             # Action Space Box(-2.0, 2.0, (1,), float32)
 
-    agent = Rock_Paper_Scissors_Agent()
+    agent = MyAgent()  # Rock_Paper_Scissors_Agent()
```

Comment:
The agent class used for the laser hocky should have a new more specific name and should not use the generic name from the beginning.

Reply:
Yeah, good observation. I changed it to have a more specific name